<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paaikkuna</class>
 <widget class="QDialog" name="Paaikkuna">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>221</width>
     <height>176</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="paivita">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Päivitä</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Talleta_rahaa">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Talleta rahaa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Nosta_rahaa">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nosta rahaa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Kirjaudu_ulos">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kirjaudu ulos</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nimi:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>136</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Saldo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nimiLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>109</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="saldoLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>136</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="tapahtumaBrowser">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>531</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Tilitapahtumat:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
